client/my-sites/plans/plan-overview/plan-features/index.jsx

```diff
@@ -81,7 +81,13 @@ const PlanFeatures = React.createClass( {
 					willBeRemoved={ willBeRemoved } />
 
 				<PlanFeature
-					description={ this.translate( 'You can live chat with our happiness engineers anytime you need.' ) }
+					description={
+						isBusiness( this.props.selectedSite.plan )
+						?
+							this.translate( 'You can live chat with our happiness engineers anytime you need.' )
+						:
+							this.translate( 'You can contact our happiness engineers anytime you need.' )
```

Comment:
> I think it should be "any time"
"anytime" is the informal usage of "any time" which is ok in this context.

> Should "live chat" be in quotes or Headline Case?
I don't think so in this case because it's being used as a verb.

> Should Happiness Engineer be capitalized?
Good catch!